.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
 
 cache:
     directories:
-        - .phpunit
+        - vendor/symfony/phpunit-bridge/bin/.phpunit
```

Comment:
updated: this is still the default because I don't see any better one, but one can overwrite it with the `SYMFONY_PHPUNIT_DIR` env var now.